CMakeLists.txt

```
@@ -18,19 +18,19 @@ ign_configure_project()
 
 # all list of citadel packages
 
-ign_find_package(ignition-common3 REQUIRED VERSION 3.4)
-ign_find_package(ignition-fuel_tools4 REQUIRED VERSION 4.1)
-ign_find_package(ignition-gazebo3 REQUIRED VERSION 3.3)
-ign_find_package(ignition-gui3 REQUIRED VERSION 3.1)
-ign_find_package(ignition-launch2 REQUIRED)
-ign_find_package(ignition-math6 REQUIRED VERSION 6.6)
-ign_find_package(ignition-msgs5 REQUIRED VERSION 5.3)
-ign_find_package(ignition-physics2 REQUIRED VERSION 2.3)
-ign_find_package(ignition-plugin1 REQUIRED)
-ign_find_package(ignition-rendering3 REQUIRED VERSION 3.1)
-ign_find_package(ignition-sensors3 REQUIRED)
-ign_find_package(ignition-tools REQUIRED)
-ign_find_package(ignition-transport8 REQUIRED)
-ign_find_package(sdformat9 REQUIRED VERSION 9.2.0)
+ign_find_package(ignition-common3 REQUIRED VERSION 3.14)
+ign_find_package(ignition-fuel_tools4 REQUIRED VERSION 4.4)
+ign_find_package(ignition-gazebo3 REQUIRED VERSION 3.9)
```

Isn't `3.11.0` the latest release?

```suggestion
ign_find_package(ignition-gazebo3 REQUIRED VERSION 3.11)
```

---

oops, @iche033 beat me to it

---

I just saw this too, see my response to Ian's comment. It looks like our Gazebo 3 releases are very broken